test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkTests.cs

```diff
@@ -40,6 +41,19 @@ public void NuGetFramework_ShortName(string input, string expected)
             Assert.Equal(expected, result);
         }
 
+        [Theory]
+        [InlineData("net5.0", ".NETFramework,Version=v5.0")]
```

Comment:
`.NETCoreApp,Version=5.0`?

Reply:
https://github.com/dotnet/designs/blame/c742ac0a589d57254926c003698c2b6652e60fdc/accepted/2020/net5/net5.md#L223

Reply:
notes from code review discussion:
 // andy: wrong call...since this is the same value as TFI? nkolev agreees

Reply:
yes, going forward with .NETCoreApp as per spec for now...as we discuss options and ramifications.